scripts/core/editor3/components/index.js

```diff
@@ -1 +1,2 @@
 export {Editor3} from './Editor3';
+export {DraftSpellchecker} from './DraftSpellchecker';
```

Comment:
Since you've started this, we might as well put all of them in here. And then just import them more easily.